modules/sdk-coin-arbeth/test/unit/transferBuilder.ts

```diff
@@ -0,0 +1,187 @@
+import assert from 'assert';
+import should from 'should';
+import { KeyPair, TransferBuilder } from '../../src';
+
+import * as testData from '../resources';
+
+describe('Eth send multi sig builder', function () {
+  const toAddress = '0x7325A3F7d4f9E86AE62Cf742426078C3755730d5';
+  const xprv =
+    'xprv9s21ZrQH143K3D8TXfvAJgHVfTEeQNW5Ys9wZtnUZkqPzFzSjbEJrWC1vZ4GnXCvR7rQL2UFX3RSuYeU9MrERm1XBvACow7c36vnz5iYyj2';
+  const key = new KeyPair({ prv: xprv }).getKeys().prv as string;
+  const amount = '100000000000000000'; // equivalent to 0.1 ether
+
+  describe('should build', () => {
+    it('native coin transfer should succeed', async () => {
+      const builder = new TransferBuilder()
+        .coin('tarbeth')
+        .expirationTime(1590078260)
+        .amount(amount)
+        .to(toAddress)
+        .contractSequenceId(2)
+        .key(key)
+        .data('0x');
+      const result = builder.signAndBuild();
+      should.equal(result, testData.SEND_FUNDS_DATA);
+    });
+
+    it('native coin transfer with coin explicitly set should succeed', async () => {
+      const builder = new TransferBuilder()
+        .expirationTime(1590078260)
+        .coin('tarbeth')
+        .amount(amount)
+        .to(toAddress)
+        .contractSequenceId(2)
+        .key(key)
+        .data('0x');
+      const result = builder.signAndBuild();
+      should.equal(result, testData.SEND_FUNDS_DATA);
```

Comment:
What is the difference between the two testcases? does the order of the coin parameter matter?

Reply:
My bad, actually this test case was present in sdk-coin-eth because for eth, we don't need to pass coin param. But these 2 are same tests only, removing one